src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw extends Area {
```

Comment:
```suggestion
public interface Drawable {
```
you should implement AreaCalculator in Figure